src/main/java/mate/academy/spring/service/impl/AuthenticationServiceImpl.java

```diff
@@ -10,18 +14,24 @@
 public class AuthenticationServiceImpl implements AuthenticationService {
     private final UserService userService;
     private final ShoppingCartService shoppingCartService;
+    private final RoleService roleService;
 
     public AuthenticationServiceImpl(UserService userService,
-                                     ShoppingCartService shoppingCartService) {
+                                     ShoppingCartService shoppingCartService,
+                                     RoleService roleService) {
         this.userService = userService;
         this.shoppingCartService = shoppingCartService;
+        this.roleService = roleService;
     }
 
     @Override
     public User register(String email, String password) {
         User user = new User();
         user.setEmail(email);
         user.setPassword(password);
+        Set<Role> roles = new HashSet<>();
+        roles.add(roleService.getRoleByName(Role.RoleName.USER));
```

Comment:
I meant to write like this `Role.RoleneName.USER.name()`. According to the task you have to pass String here not enum